applications/nrf5340_audio/src/audio/le_audio_rx.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <nrfx_clock.h>
+
+#include "nrf5340_audio_common.h"
+#include "streamctrl.h"
+#include "audio_datapath.h"
+#include "macros_common.h"
+#include "audio_system.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(le_audio_rx, CONFIG_AUDIO_DATAPATH_LOG_LEVEL);
```

Comment:
We need to discuss if this should go directly into audio_datapath.

Reply:
Should have its own log level, needs to be added to Kconfig